deliverables/ruby_monk_problems/color.rb

```diff
@@ -7,18 +7,20 @@ def initialize(r, g, b)
   end
 
   def brightness_index
-    # your code here
+    (299*r + 587*g + 114*b) / 1000   #why not @r, @g and @b
```

Comment:
You can use `@r, @g, and @b` here too.  The reason you can access these with just `r` is because of the line above

``` ruby 
attr_reader :r, :g, :b
```

which automatically defines getters for all of those instance variables